README.adoc

```diff
@@ -230,7 +230,7 @@ include::finish/system/src/main/java/io/openliberty/guides/system/SystemService.
 
 The `SystemService` class implements the `SystemServiceGrpc` class that is generated by the proto file. The four types of services defined in the proto file are implemented in this class.
 
-The [hotspot=getProperty file=0]`getProperty()` method implements the rpc unary call defined in the proto file. Use the [hotspot=pName file=0]`getPropertyName()` getter method that is generated by gRPC to retrieve the property name from the client, and store into the `pName` variable. Get and store the System property value into the [hotspot=pValue file=0]`pValue` variable. Use the the gRPC APIs to create a [hotspot=response file=0]`SystemPropertyValue` message that its type is defined in the `SystemService.proto` file. Then, send the message to the client service through the `StreamObserver` by using its [hotspot=next file=0]`onNext()` and [hotspot=complete file=0]`onComplete()` methods.
+The [hotspot=getProperty file=0]`getProperty()` method implements the rpc unary call defined in the proto file. Use the [hotspot=pPrefix file=0]`getPropertyPrefix()` getter method that is generated by gRPC to retrieve the property name from the client, and store into the `pPrefix` variable. Get and store the System property value into the [hotspot=pValue file=0]`pValue` variable. Use the the gRPC APIs to create a [hotspot=response file=0]`SystemPropertyValue` message that its type is defined in the `SystemService.proto` file. Then, send the message to the client service through the `StreamObserver` by using its [hotspot=next file=0]`onNext()` and [hotspot=complete file=0]`onComplete()` methods.
```

Comment:
This was correctish and is now incorrect. I thin getPropertyName would be better as getProperty here NOT anything to do with prefix. (i.e. my previous comment about Prefix did not apply to the singular property calls.)